lib/config/definitions.ts

```diff
@@ -883,7 +883,7 @@ const options: RenovateOptions[] = [
       'bump',
     ],
     allowString: true,
-    stage: 'package',
+    stage: 'branch',
```

Comment:
It fixes name conflict, but I guess it's potentially dangerous

Reply:
We plan to rename the package file one to be matchUpdateTypes